contrib/teamcity/build-configurations.py

```diff
@@ -553,6 +553,12 @@ def main():
             str(default_config_path)
         )
     )
+    parser.add_argument(
```

Comment:
?

Reply:
Yeah IDK. Some autoformatting happened it looks like, but why I don't know.